docs/docs/workflows/create.md

```diff
@@ -29,6 +32,51 @@ For details on the flags, consult the command help via `constellation create -h`
 
 *create* stores your cluster's state into a [`terraform.tfstate`](../architecture/orchestration.md#cluster-creation-process) file in your workspace.
 
+</tabItem>
+<tabItem value="terraform" label="Terraform">
+
+:::caution
+
+There is no extensive input validation with Terraform. If unsure, please use the Constellation CLI instead.
+
+:::
+
+Constellation also supports managing the infrastructure via Terraform. This allows for an easier GitOps integration as well as meeting regulatory requirements.
+Since the Constellation CLI also uses Terraform under the hood, the same Terraform files can be reused.
+For now, please try to refrain from changing the Terraform resource definitions, as Constellation depends on certain components.
+
+Download the Terraform files for the selected CSP from our [GitHub repository](https://github.com/edgelesssys/constellation/tree/main/cli/internal/terraform/terraform).
+
+Create a `terraform.tfvars` file.
+There, define all needed variables found in `variables.tf` using the values from the `constellation-config.yaml`.
+
+To find the image reference for your CSP and region, execute:
+
+```bash
+CONSTELL_VER=vX.Y.Z
+curl -s https://cdn.confidential.cloud/constellation/v1/ref/-/stream/stable/$CONSTELL_VER/image/info.json | jq
+```
+
+Now, initialize and apply Terraform to create the configured infrastructure:
+
+```bash
+terraform init
+terraform apply
+```
+
+The initialization depends on the already created `constellation-config.yaml` and the `constellation-id.json`.
+Therefore, create the `constellation-id.json` using the output from the Terraform state and the `constellation-conf.yaml`:
```

Comment:
```suggestion
The Constellation [init step](#the-init-step) requires the already created `constellation-config.yaml` and the `constellation-id.json`.
Create the latter using the output from the Terraform state and the `constellation-conf.yaml`:
```
I was confused because I first related "the initialization" to `terraform init`, so I suggest to clarify this a bit

Reply:
Changed `latter` to `constellation-id.json` so one does not need to backtrack to the sentence before.